### PR TITLE
OnyRamdago commit

### PR DESCRIPTION
**Pull request by OnyRamdago**

### CODE REVIEW

File: unredactor.tsv [hunk 1]
```diff
@@ -4332,4 +4332,95 @@ BhumoYangzom	testing	Sidney Franklin	██████████████
...
+OnyRamdago	testing	David Hasselhoff	On the island already are ████████████████ and his German-speaking virgin girlfriend.
+OnyRamdago	testing	Linda Blair	It starts off with pregnant ███████████, who runs down a hallways to flee what might be monsters or people with pitchforks, I'm not sure.
+OnyRamdago	testing	Linda	I guess they all traveled to the island together, but it really seemed like █████ and the boy were apart from the rest of them (maybe they were filmed separately).
+OnyRamdago	testing	Patrick Stewart	 When ███████████████ enters the scene and you think the film might get better as he brightens up the dull atmosphere.
+OnyRamdago	testing	Carly Pope	Four things intrigued me as to this film - firstly, it stars ██████████ (of "Popular" fame), who is always a pleasure to watch.
+OnyRamdago	testing	Rena Owen	Secondly, it features brilliant New Zealand actress █████████.
+OnyRamdago	testing	Adrian Pauls	I understood and can wax lyrical about the fool and found ████████████ role crucial and penetrating and then ?????
+
```
Comment (cegme) [inline unredactor.tsv:4426]: Please no empty lines

**Review by cegme — CHANGES_REQUESTED** [review]
Please resolve conflicts and resubmit a request

**Comment by OnyRamdago** [comment]
Resolved empty lines

**Comment by cegme** [comment]
Looks like there is still a confict. Please try and remove conflict, rerequest a pull request

**Comment by OnyRamdago** [comment]
2nd Conflict resolved

File: unredactor.tsv [hunk 2]
```diff
@@ -4513,6 +4513,97 @@ BhumoYangzom	testing	O-Lan	The film remains relevant today because it explores t
...
+OnyRamdago	testing	David Hasselhoff	On the island already are ████████████████ and his German-speaking virgin girlfriend.
+OnyRamdago	testing	Linda Blair	It starts off with pregnant ███████████, who runs down a hallways to flee what might be monsters or people with pitchforks, I'm not sure.
+OnyRamdago	testing	Linda	I guess they all traveled to the island together, but it really seemed like █████ and the boy were apart from the rest of them (maybe they were filmed separately).
+OnyRamdago	testing	Patrick Stewart	 When ███████████████ enters the scene and you think the film might get better as he brightens up the dull atmosphere.
+OnyRamdago	testing	Carly Pope	Four things intrigued me as to this film - firstly, it stars ██████████ (of "Popular" fame), who is always a pleasure to watch.
+OnyRamdago	testing	Rena Owen	Secondly, it features brilliant New Zealand actress █████████.
+OnyRamdago	testing	Adrian Pauls	I understood and can wax lyrical about the fool and found ████████████ role crucial and penetrating and then ?????
+=======
```
Comment (cegme) [inline unredactor.tsv:4606]: Please remove this line

**Review by cegme — CHANGES_REQUESTED** [review]

**Comment by OnyRamdago** [comment]
Removed the special characters

**Review by cegme — APPROVED** [review]

**Pull request merged**